packages/design-system/src/images/usa-banner-https.svg

```diff
@@ -1,4 +1,4 @@
-<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 54 54">
-    <path class="cls-1" fill="#719f2a" d="M34.72,34.84a1.29,1.29,0,0,1-1.29,1.29H20.57a1.29,1.29,0,0,1-1.29-1.29V27.12a1.29,1.29,0,0,1,1.29-1.29H21V23.26a6,6,0,0,1,12,0v2.57h0.43a1.29,1.29,0,0,1,1.29,1.29v7.72Zm-4.29-9V23.26a3.43,3.43,0,0,0-6.86,0v2.57h6.86Z"/>
-    <circle fill="none" stroke="#538200" stroke-miterlimit="10" class="cls-2" cx="27" cy="27.12" r="26"/>
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 54 54">
+    <path id="svg" fill="#719f2a" d="M34.72,34.84a1.29,1.29,0,0,1-1.29,1.29H20.57a1.29,1.29,0,0,1-1.29-1.29V27.12a1.29,1.29,0,0,1,1.29-1.29H21V23.26a6,6,0,0,1,12,0v2.57h0.43a1.29,1.29,0,0,1,1.29,1.29v7.72Zm-4.29-9V23.26a3.43,3.43,0,0,0-6.86,0v2.57h6.86Z"/>
```

Comment:
Theres a way to combine these two SVG elements into a single one by wrapping the two in a `<g>` element. This way we only need one id